docs/alerts-and-notifications/creating-alerts-with-netdata-alerts-configuration-manager.md

```diff
@@ -1,44 +1,111 @@
 # Creating Alerts with Netdata Alerts Configuration Manager
 
-The Netdata Alerts Configuration Manager enables subscribers to easily set up Alerts directly from the Netdata Dashboard. More details on subscriptions can be found [here](https://www.netdata.cloud/pricing/).
+The **Netdata Alerts Configuration Manager** lets you create and fine-tune alerts directly from the Netdata Cloud Dashboard.  
+To use this feature, you’ll need an active Netdata subscription. → [View subscription plans](https://www.netdata.cloud/pricing/)
 
-## Using the Alerts Configuration Manager
+---
 
-1. Navigate to the **Metrics** tab and select the chart you want to configure for Alerts.
-2. Click the **Alert icon** in the top right corner of the chart.
-3. The Alert Configuration Manager will open, showing the default thresholds. Modify these thresholds as needed; the Alert definition on the right will update automatically.
-4. For additional settings, toggle **Show advanced**.
-5. After configuring the Alert, copy the generated Alert definition from the code box. Paste this into an existing or new custom health configuration file located at `<path to netdata install>/etc/netdata/health.d/` on a Parent Agent or a Standalone Child Agent. The guide to edit health configuration files is available [here](/src/health/REFERENCE.md#edit-health-configuration-files).
-6. To activate the new Alert, run the command `<path to netdata install>/usr/sbin/netdatacli reload-health`.
+## How to Create an Alert (Step-by-Step)
 
-## Alerts Configuration Manager Sections
+1. Go to the **Metrics** tab in your Netdata Cloud Dashboard.
+2. Find the chart you want to monitor.
+3. Click the **alert icon** in the top-right corner of the chart.
+4. The **Alert Configuration Manager** will open.
+5. Adjust the threshold values. The alert definition on the right updates live as you make changes.
+6. (Optional) Click **Show advanced** for more detailed settings.
+7. Copy the generated alert definition from the code box.
+8. Paste it into a custom alert file on your Agent:
 
-### Alert Detection Method
+   ```
+   <netdata-install>/etc/netdata/health.d/your-alert.conf
```

Comment:
```suggestion
   <INSTALL_PREFIX>/etc/netdata/health.d/your-alert.conf
```

I do believe install_prefix is more appropriate here, [ref](https://learn.netdata.cloud/docs/netdata-agent/installation/linux/#destination-directory)